week1/index.js

```diff
@@ -1 +1,57 @@
-console.log("Hello world")
\ No newline at end of file
+import HTTP from 'http'
+import dflt from './resFun/dflt.js'
+import addFn from './resFun/addFn.js'
+import removeFn from './resFun/removeFn.js'
+import resetFn from './resFun/resetFn.js'
+// import Path from 'path'
+
+var state = 10
+let addOne = () => ++state
```

Comment:
I like these functions a lot! Very nice! You could have used `const addOne = () => ++state` instead of `let`, just to make sure the function value is never modified.